### PR TITLE
Smokrow/vue example

### PR DESCRIPTION
**Pull request by Smokrow**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
Currently the vue example only builds a complete webapplication.

Issue Number: N/A

## What is the new behavior?
Added a new example which shows the use of the vue-cli and webpack stand alone to compile the HelloWorld Componant into a npm package that other componants/apps can depend on.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
I ran a short speed check of the build process of the /src/components/HelloWorld:HelloWebpack (Build with Webpack) and the /src/components/HelloWorld:HelloWorld target (Build with Vue-Cli). I added the webpack example in order to show the difference between a clean and optimized "webpack" build without unnecessary plugins and the vue-cli build. When reading through the vue webpack documentation you can see that it is possible to disable plugins in the vue.config.json file ( https://cli.vuejs.org/guide/webpack.html#replacing-loaders-of-a-rule ) in order to tweak it.

Speed results:
Vue-Cli dev mode: ~5.5s critical path ~3.41s
Webpack dev mode: ~5.0 critical path ~2.41s
Vue-Cli production mode: ~ 4.019s ~3.36s
Webpack production mode: ~3.061s ~2.70s

Basically the Webpack dev mode is the "as fast as it gets" compilation while relying on the vue-loader of webpack.

### CODE REVIEW

**Comment by alexeagle** [comment]
@googlebot I consent.
